packages/prisma/zod-utils.ts

```diff
@@ -750,7 +750,8 @@ export const getRetellLLMSchema = z
           })
           .passthrough()
       )
-      .nullable(),
+      .nullable()
+      .optional(),
```

Comment:
the return type changed. earlier it used to always return null when 'states' was not present